frontend/src/components/LayoutItem.module.css

```diff
@@ -0,0 +1,37 @@
+.item {
```

Comment:
to improve UX please add some styling for .item:hover and .item:active pseudo selectors. :active is applied on click, so it should have the same styling as the .item_active class. As for hover, find a darker shade of gray compared to default, and lighter than active shade